app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:

```

Comment:
> It must return int varialbe

I don't understand what's wrong with my code. If we're talking on Player-class level, I guess None is right annotation for function. Because, if we're talking about other classes level, a could re-implement those functions (get_rating and player_info) with other functionality for each child class, that's why the annotations could be different. And I implemented that on other classes levels. That's why I don't understand what's wrong?

Reply:
For example `calculate_team_total_rating` receive a `list[Player]` where method get_rating returns None. How do you understand that we can use the sum function(if we do sum([None, None]) we will receive an error? We need to provide a base type for all abstract methods

Reply:
@ArsenPidhoretskyi Good point. @serg121192  Please make the changes